core-services/release-controller/_releases/priv/release-ocp-4.17-arm64.json

```diff
@@ -23,92 +23,6 @@
-        },
-        "aws-ovn-single-node-arm64": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64-single-node-priv"
-            }
-        },
-        "aws-ovn-techpreview-arm64": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64-techpreview-priv"
-            }
-        },
-        "aws-ovn-techpreview-serial-arm64": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64-techpreview-serial-priv"
-            }
-        },
-        "azure-ovn-arm64": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-azure-ovn-arm64-priv"
-            }
-        },
-        "azure-ovn-upgrade": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-upgrade-azure-ovn-arm64-priv"
-            },
-            "upgrade": true
-        },
-        "azure-ovn-upgrade-minor": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-stable-4.16-ocp-e2e-upgrade-azure-ovn-arm64-priv"
-            },
-            "upgrade": true,
-            "upgradeFromRelease": {
-                "candidate": {
-                    "stream": "nightly",
-                    "version": "4.16"
-                }
-            }
-        },
-        "gcp-ovn-arm64": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-gcp-ovn-arm64-priv"
-            }
-        },
-        "gcp-ovn-upgrade": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-upgrade-gcp-ovn-arm64-priv"
-            },
-            "upgrade": true
-        },
-        "gcp-ovn-upgrade-minor": {
-            "disabled": true,
-            "optional": true,
-            "prowJob": {
-                "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-stable-4.16-ocp-e2e-upgrade-gcp-ovn-arm64-priv"
-            },
-            "upgrade": true,
-            "upgradeFromRelease": {
-                "candidate": {
-                    "stream": "nightly",
-                    "version": "4.16"
-                }
-            }
-        },
         "ovn-upgrade": {
             "disabled": true,
```

Comment:
All of the existing jobs appear disabled. I thought we were leaving at least an e2e job for the P/Z/A single arch controllers. IDK we'd still want upgrades, but it's weird to me that there are no active jobs at all.